app/components/utils/back-button/template.hbs

```diff
@@ -0,0 +1,3 @@
+<div class="vlc-link-pointer u-padding--left--small" {{action "goToParentRoute"}}>
+  <li class="vl-button__icon vl-vi vl-vi-nav-left"></li>
+</div>
```

Comment:
No newline at end of file

Reply:
I noticed that in the repo, `.editorconfig` had the following:
`
[*.hbs]
insert_final_newline = false
`
Then it kinda made sense why there never was a newline. I've removed the rule in the config and now on save it will autmatically set a newline.